src/ApplicationManager.cpp

```diff
@@ -151,15 +151,18 @@ void ApplicationManager::decidePlay(world_new::World *_world) {
 
     // A new play will be chosen if the current play is not valid to keep, or the roles are all finished, in which case the
```

Comment:
There is no check anymore for the roles being finished. Maybe update the doc?

Reply:
Good find!